spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowSuspiciousConnectsModel.scala

```diff
@@ -87,33 +80,28 @@ class FlowSuspiciousConnectsModel(topicCount: Int,
       recordsWithIPTopicMixes.selectExpr(schema: _*).withColumnRenamed(TopicProbabilityMix, DstIpTopicMix)
     }
 
-    val scoreFunction = new FlowScoreFunction(timeCuts,
-      ibytCuts,
-      ipktCuts,
-      topicCount,
-      wordToPerTopicProbBC)
+
+    val scoreFunction = new FlowScoreFunction(topicCount, wordToPerTopicProbBC)
 
 
     val scoringUDF = udf((hour: Int,
-                          minute: Int,
-                          second: Int,
                           srcIP: String,
                           dstIP: String,
                           srcPort: Int,
                           dstPort: Int,
-                          ipkt: Long,
+                          protocol: String,
                           ibyt: Long,
```

Comment:
It was inconsistent throught whether ibyt was the field before ipkt or if ipkt was the field before ibyt.  This was causing some frustrating errors.... we need a good way to catch slip ups like these; the only way I know how is to have very good unit test coverage that includes test cases that hit each feature in isolation.